packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -174,6 +174,10 @@ export class AngularCompilerPlugin {
     return { path, className };
   }
 
+  get typescriptProgram(): ts.Program {
+    return this._getTsProgram();
```

Comment:
return just the typechecker here

Reply:
this can also be unavailable during certain points in the compilation lifecycle so should also return null in those cases.